Update the installation guide in order to reduce the emphasis on Anaconda (#5885)
gammapy/gammapy: docs/getting-started/usage.rst

If you haven't used Jupyter before, try typing ``print("Hello Jupyter")`` in the
first input cell, and use the keyboard shortcut ``SHIFT + ENTER`` to execute it.

> Note that one can utilise the ipykernel functionality of Jupyter Notebook to pick
> up one Gammapy version amongst your pre-installed ones (see :ref:`quickstart-setup`).

Reviewer: ```suggestion
Note that one can utilise the ipykernel functionality of Jupyter Notebook to select 
a specific pre-installed Gammapy version from your system (see :ref:`quickstart-setup`).
```
